libs/mrwebrtc/src/media/audio_track_read_buffer.h

```diff
@@ -19,11 +20,12 @@ struct AudioFrame;
 class PeerConnection;
 
 /// Implementation of |mrsAudioTrackReadBufferHandle|.
-class AudioTrackReadBuffer : public webrtc::AudioTrackSinkInterface {
+class AudioTrackReadBuffer : public TrackedObject, webrtc::AudioTrackSinkInterface {
```

Comment:
Why is this needed?

Reply:
The destructor calls a WebRTC function and needs the signaling thread to be alive at that point.

Reply:
It's unfortunate we (or rather, I) mixed the concepts of ref-counted and tracked into the same class 😢